test/unit/component/parser/json/BpmnJsonParser.startEvent.test.ts

```diff
@@ -146,4 +146,63 @@ describe('parse bpmn as json for start event', () => {
       boundsHeight: 46,
     });
   });
+
+  it('json containing one process with an array of start events, some are not NONE event', () => {
+    const json = `{
+  "definitions": {
+    "process": {
+      "startEvent": [
+        { "id": "event_id_0", "name": "none start event" },
+        { "id": "event_id_1", "conditionalEventDefinition": {} },
```

Comment:
:information_source: I put only event defintiions that are supported in BPMN for Start Event even if the code can manage all cases